CODEOWNERS

```diff
@@ -1090,6 +1090,8 @@ build.json @home-assistant/supervisor
 /tests/components/soundtouch/ @kroimon
 /homeassistant/components/spaceapi/ @fabaff
 /tests/components/spaceapi/ @fabaff
+/homeassistant/components/spaceapi1/ @fabaff
```

Comment:
@fabaff was this change on purpose? Seems unrelated to me and component spaceapi1 does not exist

Reply:
No, accidentally used `-a`.